pkg/model/pki.go

```diff
@@ -133,6 +133,16 @@ func (b *PKIModelBuilder) Build(c *fi.ModelBuilderContext) error {
 
 	{
 		t := &fitasks.Keypair{
+			Name:      fi.String("kube-proxy-client"),
```

Comment:
Actually, this is going to be a confusing name I've realized...  When I look at it, I'm going to assume something to do with kube-proxy :-)

How about `apiserver-proxy-client` ?